crypto/aes_openpgp.go

```diff
@@ -17,24 +18,52 @@ type AESOpenPGPClient struct {
 }
 
 type AESOpenPGPSettings struct {
+	Cipher   string `json:"cipher,omitempty" toml:"cipher"`
+	S2KCount int    `json:"s2kcount,omitempty" toml:"s2kcount"`
 }
 
 func DefaultAESOpenPGPSettings() *AESOpenPGPSettings {
-	return &AESOpenPGPSettings{}
+	return &AESOpenPGPSettings{
+		Cipher:   "aes256",
+		S2KCount: 65011712,
+	}
 }
 
 func NewAESOpenPGPClient(config Config, settings AESOpenPGPSettings) (*AESOpenPGPClient, error) {
 	c := &AESOpenPGPClient{
 		config:   config,
 		settings: settings,
 	}
-	c.packetConfig = &packet.Config{}
+	c.packetConfig = &packet.Config{
+		DefaultCipher: c.cipherFunc(),
+		S2KCount:      c.s2kCount(),
+	}
 	return c, nil
 }
 
+func (c *AESOpenPGPClient) cipherFunc() packet.CipherFunction {
+	switch c.settings.Cipher {
+	default:
```

Comment:
Is it required to specify a Cipher in the config? I'd expect if I only specify `type = "aes_openpgp"`, the default cipher would be used.